apps/src/templates/sectionsRefresh/SectionsSetUpContainer.jsx

```diff
@@ -258,6 +262,30 @@ export default function SectionsSetUpContainer({
     );
   };
 
+  const renderChildAccountPolicyNotification = () => {
+    const isEmailLoggin = queryParams('loginType') === 'email';
+    const isStudentSection = queryParams('participantType') === 'student';
+    // We want to display a Child Account Policy warning notification for US
```

Comment:
This comment feels more appropriate for where we are setting `showChildAccountPolicy` since this logic does not check whether a teacher is a US teacher, just whether the flag is true.

Reply:
I moved all logic to the react component, passing the `userCountry` as a prop.
I don't think this is ideal, but I'm choosing to have all logic localized in the same place.